draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -180,6 +188,10 @@ This again allows the provider to query only for application-specific records it
 
 Consumers of the provider services need to relay information from a provider's website to their local DNS administrators. The exact DNS record type, content and location is often not clear when the DNS administrator receives the information, especially to consumers who are not DNS experts. Providers SHOULD offer detailed help pages, that are accessible without needing a login on the provider website, as the DNS adminstrator often has no login account on the provider service website. Similarly, for clarity, the exact and full DNS record (including a Fully Qualified Domain Name) to be added SHOULD be provided along with help instructions.
 
+
+TODO: Should the format of the verifying record be enhanced to include an indication of scope: e.g. single hostname vs entire domain? This could be done in a number of ways, e.g adding a "wildcard" substring or label into the DNS record owner name, or by specifying a wildcard attribute in the TXT RDATA string.
```

Comment:
I think the need for metadata within the record itself is less here because as long as the service provider is clear about what the scope of the record is, it will grant the privilege accordingly. The domain owner (user) wouldn't really benefit from having this information around in their DNS.

Reply:
It depends on the details of the service, though, and what kind of privilege it is granting doesn't it?. For the certificate case for example, the domain owner would definitely want to know whether its for a wildcard certificate, which the domain owner may not want to grant. For other types of services, it may or may not matter. But yes, this info could also be communicated in the documentation of the service. Let me pull out this TODO: paragraph, until we've talked through this issue.